tests/components/bmw_connected_drive/__init__.py

```diff
@@ -43,12 +47,29 @@
 async def setup_mocked_integration(hass: HomeAssistant) -> MockConfigEntry:
     """Mock a fully setup config entry and all components based on fixtures."""
 
+    # Mock config entry and add to HA
     mock_config_entry = MockConfigEntry(**FIXTURE_CONFIG_ENTRY)
     mock_config_entry.add_to_hass(hass)
 
     assert await hass.config_entries.async_setup(mock_config_entry.entry_id)
     await hass.async_block_till_done()
 
+    # Get the entity registry and enable all entities
+    entity_registry = er.async_get(hass)
+
+    for entity in entity_registry.entities.values():
+        if entity.disabled and entity.platform == BMW_DOMAIN:
+            entity_registry.async_update_entity(entity.entity_id, disabled_by=None)
+    await hass.async_block_till_done()
+
+    # Wait for reload time to pass
+    async_fire_time_changed(
```

Comment:
We prefer to use `freezer.tick` to move time, example: https://github.com/home-assistant/core/blob/dev/tests/components/accuweather/test_sensor.py#L63

Reply:
Not applicable anymore but I'll try to remember next time, thanks!